logger.go

```diff
@@ -160,6 +165,11 @@ func (l *Logger) output(s severity, depth int, txt string) {
 func (l *Logger) Close() {
 	logLock.Lock()
 	defer logLock.Unlock()
+
+	if !l.initialized {
```

Comment:
I think you want an explicit nil check as well because otherwise you can end up with a crash on the nil receiver. I.e. "if l == nil || !l.initialized {}"

Reply:
I'm a fan of belts and suspenders for sure.

It's not (w/o violating the API) possible to hit the nil receiver case, since either the `l` is the defaultLogger,
which is constructed in Init, or a new logger. Note that currently this would explode on the nil receiver case
anyhow, see line 173.

I can add this tho, thoughts?